lib/regular-update.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const request = require('request');
+
+// Map from URL to { timestamp: last fetch time, interval: in milliseconds,
+// data: data }.
+let regularUpdateCache = Object.create(null);
```

Comment:
I think we're on a new enough version of Node to use `Map` here, rather than an "object as a map".